src/common/btree.c

```diff
@@ -953,6 +953,9 @@ btr_check_availability(struct btr_context *tcx, struct btr_check_alb *alb)
 	if (rc == -DER_INPROGRESS) /* Unceration */
 		return PROBE_RC_INPROGRESS;
 
+	if (rc == -DER_DATA_LOSS)
```

Comment:
Not really this patch, but just noticed the that Uncertain is not spelled correctly.  Not sure why our tools haven't flagged that one.

Reply:
I will fix it when refresh this patch or in subsequent patch.